dotCMS/src/main/java/com/dotcms/util/transform/TransformerLocator.java

```diff
@@ -0,0 +1,127 @@
+package com.dotcms.util.transform;
+
+import com.dotcms.util.ReflectionUtils;
+import com.dotmarketing.beans.Identifier;
+import com.dotmarketing.beans.transform.IdentifierTransformer;
+import com.dotmarketing.portlets.containers.model.Container;
+import com.dotmarketing.portlets.containers.transform.ContainerTransformer;
+import com.dotmarketing.portlets.folders.model.Folder;
+import com.dotmarketing.portlets.folders.transform.TemplateTransformer;
+import com.dotmarketing.portlets.links.model.Link;
+import com.dotmarketing.portlets.links.transform.LinkTransformer;
+import com.dotmarketing.portlets.templates.model.Template;
+import com.dotmarketing.portlets.templates.transform.FolderTransformer;
+import com.dotmarketing.util.Config;
+import com.dotmarketing.util.Logger;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.util.Logger'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)